client/src/components/Game/index.js

```diff
@@ -88,7 +88,7 @@ class Game extends Component {
                 }
               </AuthUserContext.Consumer>
             } else {
-              return <Endgame />
+              return <FabricContextProvider />
```

Comment:
Why are you returning FabricContextProvider here? Wouldn't make more sense to use a .Consumer component instead of rendering .Provider components every time?

Reply:
https://github.com/fabricjs/fabric.js/issues/5951#issuecomment-621810684
Provides context so sub components have access to the fabric context without passing props around. Context cannot be consumed without a provider. It is being consumed here https://github.com/googleinterns/step119-2020/blob/a491eda5bd180dd0df10f9c31b632637cb836278/client/src/components/Game/FabricCanvas.js#L7. As a functional component, useContext is used instead of Consumer.

Reply:
Ah I see, although you can do it in a slightly cleaner way if you extract the logic into a higher-order component similar to the code in withAuthentication.js. That way you can just wrap the canvas in that.